utils/db/teams.js

```diff
@@ -0,0 +1,94 @@
+import shortid from "shortid";
+
+import firebase from "../firebase/firebase";
+import { setCurrentChallenge } from "./challenges";
+import { getCurrentValue } from "./helper";
+
+async function getTeamIdFromTeamCode(teamCode) {
+  return getCurrentValue(`/teamCodes/${teamCode}`);
+}
+
+export async function createTeam(teamName, userId) {
+  const team = {
+    name: teamName,
+    score: 0,
+    users: [userId]
+  };
+
+  // Create team
+  let teamRef = await firebase
+    .database()
+    .ref("/teams/")
+    .push(team);
+
+  let teamId = teamRef.key;
+
+  // Use the last six digits of team id for the team code
+  let teamCode = teamId.slice(-6);
```

Comment:
Is there a guarantee from Firebase that the last 6 digits will always be the same?

Reply:
A little confused on this. There is no guarantee that the last six digits will be unique but this is unlikely. I still have an extra check that the team code doesn't already exist. The team code is added to a team code lookup (maps team code to team id since there's no way to get the team id back unless it's in the database)

Reply:
I was just concerned we'd have collisions with identical teamCodes for different teams, but if there's a check for that already then this should be fine